Userland/Libraries/LibWeb/Layout/FormattingContext.cpp

```diff
@@ -224,8 +224,19 @@ OwnPtr<FormattingContext> FormattingContext::layout_inside(Box const& child_box,
         return {};
 
     auto independent_formatting_context = create_independent_formatting_context_if_needed(m_state, child_box);
-    if (independent_formatting_context)
+    if (independent_formatting_context) {
         independent_formatting_context->run(child_box, layout_mode, available_space);
+        
+        // For input box layout invalidation fast path
+        if(is<BlockContainer>(child_box)) {
+            // TODO: check that child box is for an HTMLInputElement before saving run info
+            // It's going to be the node associated with the box, so something like
+            // child_box.dom_node().is_html_input_element()
+
+            if(layout_mode == LayoutMode::Normal)
+                static_cast<BlockContainer&>(const_cast<Box&>(child_box)).store_layout_inside_run_info(&m_state, available_space);
+        }
+    }
```

Comment:
This section needs `clang-format` run on it. (That's what the "Lint Code" CI job is complaining about.)